attribute-utils/src/main/java/org/hypertrace/attribute/db/DbAttributeUtils.java

```diff
@@ -0,0 +1,154 @@
+  private static final String OTEL_NET_TRANSPORT = OTelDbAttributes.NET_TRANSPORT.getValue();
+
+  // mongo specific attributes
+  private static final String OTEL_MONGO_DB_SYSTEM_VALUE = OTelDbAttributes.MONGODB_DB_SYSTEM_VALUE.getValue();
+  private static final String OTHER_MONGO_ADDRESS = RawSpanConstants.getValue(Mongo.MONGO_ADDRESS);
+  private static final String OTHER_MONGO_URL = RawSpanConstants.getValue(Mongo.MONGO_URL);
+  private static final String OTHER_MONGO_OPERATION = RawSpanConstants.getValue(Mongo.MONGO_OPERATION);
+  private static final String OTHER_MONGO_NAMESPACE = RawSpanConstants.getValue(Mongo.MONGO_NAMESPACE);
+  private static final String OTEL_MONGO_COLLECTION = OTelDbAttributes.MONGODB_COLLECTION.getValue();
+
+  // redis specific attributes
+  private static final String OTHER_REDIS_CONNECTION = RawSpanConstants.getValue(Redis.REDIS_CONNECTION);
+  private static final String OTEL_REDIS_DB_SYSTEM_VALUE = OTelDbAttributes.REDIS_DB_SYSTEM_VALUE.getValue();
+
+  // sql specific attributes
+  private static final String[] OTEL_SQL_DB_SYSTEM_VALUES =
+      {
+          OTelDbAttributes.MYSQL_DB_SYSTEM_VALUE.getValue(), OTelDbAttributes.ORACLE_DB_SYSTEM_VALUE.getValue(),
+          OTelDbAttributes.MSSQL_DB_SYSTEM_VALUE.getValue(), OTelDbAttributes.DB2_DB_SYSTEM_VALUE.getValue(),
+          OTelDbAttributes.POSTGRESQL_DB_SYSTEM_VALUE.getValue(), OTelDbAttributes.REDSHIFT_DB_SYSTEM_VALUE.getValue(),
+          OTelDbAttributes.HIVE_DB_SYSTEM_VALUE.getValue(), OTelDbAttributes.CLOUDSCAPE_DB_SYSTEM_VALUE.getValue(),
+          OTelDbAttributes.HSQLDB_DB_SYSTEM_VALUE.getValue(), OTelDbAttributes.OTHER_SQL_DB_SYSTEM_VALUE.getValue()
+      };
+  private static final String JDBC_EVENT_PREFIX = "jdbc";
+  private static final String SQL_URL = RawSpanConstants.getValue(Sql.SQL_SQL_URL);
+
+  public static Optional<String> getMongoURI(Event event) {
+    if (SpanAttributeUtils.containsAttributeKey(event, OTHER_MONGO_ADDRESS)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, OTHER_MONGO_ADDRESS));
+    } else if (SpanAttributeUtils.containsAttributeKey(event, OTHER_MONGO_URL)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, OTHER_MONGO_URL));
+    } else if (SpanAttributeUtils.containsAttributeKey(event, OTEL_DB_SYSTEM)) {
+      if (!OTEL_MONGO_DB_SYSTEM_VALUE.equals(SpanAttributeUtils.getStringAttribute(event, OTEL_DB_SYSTEM))) {
+        return Optional.empty();
+      }
+      return getBackendURIForOtelFormat(event);
+    }
+
+    return Optional.empty();
+  }
+
+  public static List<String> getAttributeKeysForMongoOperation() {
+    return Lists.newArrayList(Sets.newHashSet(OTHER_MONGO_OPERATION, OTEL_DB_OPERATION));
+  }
+
+  public static List<String> getAttributeKeysForMongoNamespace() {
+    return Lists.newArrayList(Sets.newHashSet(OTEL_MONGO_COLLECTION, OTHER_MONGO_NAMESPACE));
```

Comment:
```suggestion
    return Lists.newArrayList(Sets.newHashSet(OTHER_MONGO_NAMESPACE, OTEL_MONGO_COLLECTION));
```

Reply:
Let's keep the order consistent - other first, then otel as in previous method `getAttributeKeysForMongoOperation`

Reply:
done